src/index.css

```diff
@@ -11,3 +11,221 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+/* font-family: 'Poppins', sans-serif; */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500&display=swap');
```

Comment:
Great job of importing fonts here! Maybe next time, try adding in index.html in public folder